src/frontend-admin/src/services/AxiosService/AxiosService.ts

```diff
@@ -0,0 +1,135 @@
+import axios, {AxiosInstance} from 'axios'
+//import qs from 'qs' ??
+//import buildQuery from 'odata-query' ??
```

Comment:
This one is an interesting question mark to me - I know Joe has expressed interest in using OData but I don't think I've seen it implemented in the backend yet. We'll see